.vscode/settings.json

```diff
@@ -86,6 +86,7 @@
     "xtr1common": "cpp",
     "xtree": "cpp",
```

Comment:
remove modifications from this file thanks.

Reply:
@realstealthninja removed the modifications from this file.